governance/votes/Executive vote - March 26, 2024.md

```diff
@@ -0,0 +1,151 @@
+
+- Decrease the ETH-A SF by 2.00% from 15.25% to **13.25%**.
+- Decrease the ETH-B SF by 2.00% from 15.75% to **13.75%**.
+- Decrease the ETH-C SF by 2.00% from 15% to **13%**.
+- Decrease the WSTETH-A SF by 2% from 16.25% to **14.25%**.
+- Decrease the WSTETH-B SF by 2% from 16% to **14%**.
+- Decrease the WBTC-A SF by 2% from 16.75% to **14.75%**.
+- Decrease the WBTC-B SF by 2% from 17.25% to **15.25%**.
+- Decrease the WBTC-C SF by 2% from 16.50% to **14.50%**.
+
+### SparkLend [D3M](https://manual.makerdao.com/module-index/module-dai-direct-deposit) Update
+
+- **Authorization:** [Forum post](https://vote.makerdao.com/polling/QmVGDsvm)
+- **Proposal:** [Forum post](http://forum.makerdao.com/t/mar-6-2024-proposed-changes-to-sparklend-for-upcoming-spell/23791)
+
+If this executive proposal passes, the following change to the SparkLend [D3M](https://manual.makerdao.com/module-index/module-dai-direct-deposit) [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be made:
+
+- Increase the SparkLend D3M Maximum Debt Ceiling by 1.0 billion DAI from 1.5 billion DAI to **2.5 billion DAI**.
+
+### [D3M](https://manual.makerdao.com/module-index/module-dai-direct-deposit) to Spark DAI Morpho Vault (DIRECT-SPARK-MORPHO-DAI)
+
+- **Authorization:** [Forum post](https://forum.makerdao.com/t/introduction-and-initial-parameters-for-ddm-overcollateralized-spark-metamorpho-ethena-vault/23925/5)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/introduction-and-initial-parameters-for-ddm-overcollateralized-spark-metamorpho-ethena-vault/23925)
+
+If this executive proposal passes, the D3M will be deployed to the [Spark DAI Morpho Vault](https://morpho.blockanalitica.com/metamorpho/vaults/0x73e65dbd630f90604062f6e02fab9138e713edd9) with the following parameters:
+
+#### [D3M](https://manual.makerdao.com/module-index/module-dai-direct-deposit) [DC-IAM](https://manual.makerdao.com/module-index/module-dciam) Parameters
+
+- [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line): 100 million DAI
+- [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap): 100 million DAI
+- [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl): 24 hours
+- Debt Write-off Timelock (`tau`) (Defines the time to elapse before debt can be written off after a D3M has been caged): 7 days
+
+#### [D3M](https://manual.makerdao.com/module-index/module-dai-direct-deposit) Addresses
+
+- oracle: 0xA5AA14DEE8c8204e424A55776E53bfff413b02Af
+- plan: 0x374b5f915aaED790CBdd341E6f406910d648fD39
+- pool: 0x9C259F14E5d9F35A0434cD3C4abbbcaA2f1f7f7E
+- operator: 0x298b375f24CeDb45e936D7e21d6Eb05e344adFb5
+
+#### Additional Actions
+
+If this executive proposal passes, the DIRECT_MOM Breaker will be extended to include the Morpho D3M.
```

Comment:
You might want to include an explanation of what this enables. For instance, a brief note such as "enabling governance to deactivate vault functionality through an executive spell without needing to wait for the GSM Pause Delay."